f/radio/fn_radioCheckChannels.sqf

```diff
@@ -0,0 +1,89 @@
+// F3 - Folk ARPS Long-Range Radio Module
+// Credits and documentation: https://github.com/folkarps/F3/wiki
+
+/* ========================
+This function checks what channels a player should have access to. It's called by event handlers set up by fn_radioAddHandlers.sqf.
+=========================== */
+
+// Clients wait for the server to finish with setup
+waitUntil {!(isNil "f_var_radioChannelsUnified")};
+
+_splitMode = f_var_radioSplitMode;
+
+// Initialise variables
+private _radioChannelsPlayerSpecific = [];
+private _radioChannelsVehicleSpecific = [];
+private _channelChecked = 0;
+private _channelBackpacks = [];
+private _channelsToAddListen = [];
+private _channelsToAddTalk = [];
+
+for "_i" from 1 to 10 do {
+
+	_channelChecked = _i;
```

Comment:
i would get rid of `_channelChecked` or `_i` and only use one of them

Reply:
Fixed